src/knexfile.ts

```diff
@@ -4,11 +4,11 @@ module.exports = {
   development: {
     client: 'mysql2',
     connection: {
-      host: '127.0.0.1',
-      port: 3306,
-      user: 'tt',
-      password: 'password',
-      database: 'tt',
+      host: process.env.MYSQL_HOST,
```

Comment:
Another thing that's worth testing is migrations. Do they run?

Reply:
they do not! hmmm